dali/operators/reader/loader/video_loader.cc

```diff
@@ -222,7 +222,12 @@ static int read_packet(void *opaque, uint8_t *buf, int buf_size) {
     int ret = fseek(file_data->file_stream, file_data->file_position, SEEK_SET);
     DALI_ENFORCE(ret == 0, make_string("Could not open file ", file_data->filename));
   }
-  return fread(buf, 1, buf_size, file_data->file_stream);
+  auto ret = fread(buf, 1, buf_size, file_data->file_stream);
```

Comment:
Can you check for actual EOF: https://en.cppreference.com/w/cpp/io/c/feof
This is can be slightly different case than getting ret != buf_size.

Reply:
It is not `ret != buf_size` , it is `ret == 0`. But added just in case.

Reply:
Cool 👍